backend/src/test/java/com/huellapositiva/api/ProposalControllerShould.java

```diff
@@ -84,11 +85,94 @@
     @Autowired
     private ProposalRepository proposalRepository;
 
+    @Autowired
+    private ProposalService proposalService;
+
     @BeforeEach
     void beforeEach() {
         testData.resetData();
     }
 
+    @Test
+    void change_status_to_finished() throws Exception {
+        // GIVEN
+        JpaProposal publishedProposal= testData.registerESALAndProposal(PUBLISHED);
+        publishedProposal.setClosingProposalDate(new SimpleDateFormat("dd-MM-yyyy").parse("20-12-2020"));
+
+        // WHEN
+        proposalService.changeStatusToFinished(publishedProposal.getId());
```

Comment:
This test should be moved inside ProposalService

XXControllerShould is testing API requests involving HTTP request/responses